pkg/apis/rollouts/v1alpha1/analysis_types.go

```diff
@@ -378,6 +388,9 @@ type AnalysisRunSpec struct {
 	// +patchStrategy=merge
 	// +optional
 	MeasurementRetention []MeasurementRetention `json:"measurementRetention,omitempty" patchStrategy:"merge" patchMergeKey:"metricName" protobuf:"bytes,5,rep,name=measurementRetention"`
+	// TtlStrategy object contains the strategy for the time to live depending on if the analysis succeeded or failed
+	// +optional
+	TtlStrategy *TtlStrategy `json:"ttlStrategy,omitempty" protobuf:"bytes,6,opt,name=ttlStrategy"`
```

Comment:
Same comment here, `TTLStrategy` for the field name.

Reply:
Done